bootstrap/handlers/httpserver.go

```diff
@@ -98,7 +98,7 @@ func (b *HttpServer) BootstrapHandler(
 
 	timeout, err := time.ParseDuration(bootstrapConfig.Service.RequestTimeout)
 	if err != nil {
-		lc.Errorf("unable to parse RequestTimeout value of %s to a duration: %w", bootstrapConfig.Service.RequestTimeout, err)
+		lc.Errorf("unable to parse RequestTimeout value of %s to a duration: %s", bootstrapConfig.Service.RequestTimeout, err.Error())
```

Comment:
The common practice is 
```suggestion
		lc.Errorf("unable to parse RequestTimeout value of %s to a duration: %v", bootstrapConfig.Service.RequestTimeout, err.Error())
```

Reply:
Fix applied.  Please give another look.